modules/extensions-xml-support/src/test/resources/modules/module-json-custom-types-catalog.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<types:mule xmlns="http://www.mulesoft.org/schema/mule/core"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:types="http://www.mulesoft.org/schema/mule/types">
+<catalogs xmlns="http://www.mulesoft.org/schema/mule/types" >
 
-    <types:catalog>
-        <types:type name="Type1" location="./module-json-custom-types-type1-schema.json"/>
-        <types:type name="Type2" location="./module-json-custom-types-type2-schema.json"/>
-    </types:catalog>
+    <catalog name="Type1" format="application/json">
+        <schema format="application/json+schema" location="./module-json-custom-types-type1-schema.json" />
```

Comment:
>So I would be able to do: MY_JSON_SCHEMA_HERE or do:
${file::MY_JSON_SCHEMA_LOCATION_HERE}

we could test it, but probably won't work for the types.. it should work for the rest of the `<module>` though.. as smart connectors relies in the runtime to load the XMLs ( org.mule.runtime.config.spring.XmlConfigurationDocumentLoader )